packages/services/f-stat-client/src/statClient.js

```diff
@@ -0,0 +1,55 @@
+const { Client } = require('@elastic/elasticsearch');
+
+export default class StatClient {
+    constructor (
+        url,
+        port,
+        tenant,
+        featureName,
+        user = null,
+        pwd = null,
+        indexName = 'justeat',
+        mock = null
+    ) {
+        const conn = {
+            node: `${url || 'http://localhost'}:${port || 9200}`
+        };
+
+        if (user) {
+            conn.auth = {
+                username: user,
+                password: pwd
+            };
+        }
+
+        if (mock) {
+            conn.Connection = mock.getConnection();
+        }
+
+        this.client = new Client(conn);
+        this.tenant = tenant || 'ns';
+        this.feature = featureName || 'NotSpecified';
```

Comment:
I would probably just default this at the top as `featureName = 'Generic Front End'